commonknowledge/mapped
# fix: add get_record_url for ActionNetworkSource to fix viewing map points

## Description
Implements missing but required functionality for ActionNetworkSource: getting the URL of a record.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-264/view-activist-names-bug

## How Can It Be Tested?
Import an Action Network member list, add it to a map report, and check the points can be clicked + viewed and the links to them work.

## How Will This Be Deployed?
Normal, but need to insert some values into the database for the `group_slug` field – `common-knowledge`, `second-group-2`, and one for a user that I will not write here.
